packages/roosterjs-editor-plugins/lib/plugins/Paste/wordConverter/commentsRemoval.ts

```diff
@@ -83,4 +84,11 @@ export default function commentsRemoval(elementCallbacks: ElementCallbackMap) {
         }
         return true;
     });
+
+    /**
+     * Remove comment related styles
+     */
+    chainSanitizerCallback(styleCallbacks, MSO_COMMENT_REFERENCE, () => false);
+    chainSanitizerCallback(styleCallbacks, MSO_COMMENT_DATE, () => false);
+    chainSanitizerCallback(styleCallbacks, MSO_COMMENT_PARENT, () => false);
```

Comment:
What are the issues if we don't remove them?

Reply:
If we do not remove this styles, when we send the email and a Win32 user receives the email, they see some style on the word that contain the comment.

For example, this is how a win32 user see a message:
![image](https://user-images.githubusercontent.com/8291124/176562471-3dc9321a-ec6a-43dd-8d7f-ec84779f2884.png)

vs how we sent it:
![image](https://user-images.githubusercontent.com/8291124/176562601-6f789278-152a-4a3c-a93a-fca07c98ad4a.png)

Reply:
got it, thanks. Better add some comment